src/aws-cli/orb.yml

```diff
@@ -0,0 +1,56 @@
+version: 2.1
+description: "Installs a ready to go awscli"
+
+executors:
+  default:
+    description: The debian based docker container to use when running the AWS CLI
+    parameters:
+      python-version:
+        type: string
+        default: "2.7"
+      debian-release:
+        type: string
+        default: "stretch"
+    docker:
+      - image: circleci/python:<< parameters.python-version >>-<< parameters.debian-release >>
+
+commands:
+  install:
+    description: |
+      Install the AWS CLI using pip.
+    steps:
+      - run:
+          name: Install AWS CLI
+          command: |
+            if [[ $(command -v aws) == "" ]]; then
+              sudo pip install awscli
+            else
+              echo "AWS CLI is already installed."
+            fi
+  configure:
+    description: |
+      Apply configure to `~/.aws/credentials` and `~/.aws/config`
+    parameters:
+      profile-name:
+        description: Profile name to be configured.
+        type: string
+        default: "default"
+      aws-access-key-id:
+        description: AWS access key id for IAM role, preferably passed in as an ENV var.
```

Comment:
Oh this is pretty cool but subtle. I was wondering how [this build](https://circleci.com/gh/circleci/aws-code-deploy-orb-test/28#config/containers/0) got the ENV vars populated, but then I realized the parameter value is a string representing a shell variable.

I think one of these descriptions should spell this out more explicitly (to preempt support tickets): "Set this to the name of the environment variable you will set to hold this value, i.e. $AWS_ACCESS_KEY.  It won't be expanded in the CircleCI UI. The shell will do that at build execution time."

Reply:
Also I think both id and secret access key should have sensible defaults, like `$AWS_SECRET_ACCESS_KEY`